static/manager-dashboard/src/Components/Performance.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { invoke } from '@forge/bridge';
+import PieChart from '../shared/PieChart';
+
+function Performance() {
+    const [selfAssessedPerformanceData, setSelfAssessedPerformanceData] = useState(null);
+    const [peerAssessedPerformanceData, setPeerAssessedPerformanceData] = useState(null);
+
+    useEffect(() => {
+        invoke('getSelfAssessedPerformanceRatings', { example: 'my-invoke-variable' }).then(setSelfAssessedPerformanceData);
+        invoke('getPeerAssessedPerformanceRatings', { example: 'my-invoke-variable' }).then(setPeerAssessedPerformanceData);
+    }, []);
+
+    const performancePieChartLabels = ['Bad', 'Somewhat Bad', 'Okay', 'Somewhat Good', 'Good'];
+    const selfPerformancePieChartTitle = 'Self Assessed Performance Ratings';
+    const peerPerformancePieChartTitle = 'Peer Assessed Performance Ratings';
+
+    return (
+        <div>
```

Comment:
Sorry for some reason this comment didn't get added in my previous Review:

To make the Pie charts better fit in the page I added this to the `div` element for satisfaction, you can try something similar either in this PR or the next one you intend to make for the UI:
```
style="margin: auto; padding: 10px; width: 50%; height: 50%"
```

Reply:
I'm gonna have a separate PR for UI update for performance page. Going to merge this PR in since it is large enough already. I'll look into this code though when updating the UI for performance!